### PR TITLE
Update design per GUI changes

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/jujusvg/54/
Test PASSed.

**Comment by makyo** [comment]
# QA
- tests pass
- `go run examples/generatesvg.go examples/mediawiki-scalable.yaml  > bundle.svg`
- View bundle.svg in a browser and ensure that it shows the new design
- compare it to the GUI: 
  juju_admin

File: assets/relation-icon-healthy.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
+package assets
+
+var RelationIconHealthy = `
+<svg
```
Comment (anthonydillon) [inline assets/relation-icon-healthy.go]: A optimised version of this SVG:

```
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16.000017 16.000017"><g transform="translate(-952 -156.362)"><path color="#000" overflow="visible" fill="none" d="M952 156.362h16v16h-16z"/><circle r="7.25" cy="164.362" cx="960" color="#000" overflow="visible" fill="#38b44a" stroke="#38b44a" stroke-width="1.5" stroke-dashoffset=".8"/><path style="line-height:125%;-inkscape-font-specification:Ubuntu;text-align:center" d="M963.8 161.286l-.066.057L959 165.49l-2.776-2.38-.84.948 3.616 3.804 5.5-5.787-.7-.79z" font-size="15" font-family="Ubuntu" letter-spacing="0" word-spacing="0" text-anchor="middle" fill="#fff"/></g></svg>
```

File: assets/relation-icon-healthy.svg [hunk 2]
```diff
@@ -1,13 +1,104 @@
...
-		<path fill="#38B44A" d="M10,3c3.859,0,7,3.14,7,7c0,3.859-3.141,7-7,7c-3.86,0-7-3.141-7-7C3,6.14,6.14,3,10,3 M10,0
-			C4.477,0,0,4.477,0,10c0,5.522,4.477,10,10,10c5.522,0,10-4.478,10-10C20,4.477,15.522,0,10,0L10,0z"/>
-	</g>
-	<circle fill="#38B44A" cx="10" cy="10" r="5"/>
-</g>
-</svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
```
Comment (anthonydillon) [inline assets/relation-icon-healthy.svg]: An optimised version of this SVG:

``` svg
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16.000017 16.000017"><g transform="translate(-952 -156.362)"><path color="#000" overflow="visible" fill="none" d="M952 156.362h16v16h-16z"/><circle r="7.25" cy="164.362" cx="960" color="#000" overflow="visible" fill="#38b44a" stroke="#38b44a" stroke-width="1.5" stroke-dashoffset=".8"/><path style="line-height:125%;-inkscape-font-specification:Ubuntu;text-align:center" d="M963.8 161.286l-.066.057L959 165.49l-2.776-2.38-.84.948 3.616 3.804 5.5-5.787-.7-.79z" font-size="15" font-family="Ubuntu" letter-spacing="0" word-spacing="0" text-anchor="middle" fill="#fff"/></g></svg>
```

File: assets/relation-icon-healthy.svg [hunk 3]
```diff
@@ -1,13 +1,104 @@
...
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
```
Comment (anthonydillon) [inline assets/relation-icon-healthy.svg]: An optimised version of this SVG:
```
svg<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16.000017 16.000017"><g transform="translate(-952 -156.362)"><path color="#000" overflow="visible" fill="none" d="M952 156.362h16v16h-16z"/><circle r="7.25" cy="164.362" cx="960" color="#000" overflow="visible" fill="#38b44a" stroke="#38b44a" stroke-width="1.5" stroke-dashoffset=".8"/><path style="line-height:125%;-inkscape-font-specification:Ubuntu;text-align:center" d="M963.8 161.286l-.066.057L959 165.49l-2.776-2.38-.84.948 3.616 3.804 5.5-5.787-.7-.79z" font-size="15" font-family="Ubuntu" letter-spacing="0" word-spacing="0" text-anchor="middle" fill="#fff"/></g></svg>
```

**Comment by anthonydillon** [comment]
Hey @makyo, it looks great! I run SVGs through this site to get nicely optimised.
https://jakearchibald.github.io/svgomg/

**Comment by makyo** [comment]
@anthonydillon awesome, thanks.  Those may go away, as I've been informed that the relations are now gray.  Whoops.  Will redo as needed.

**Comment by anthonydillon** [comment]
Yea I just went into the relation icons SVG and changed the hex.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/jujusvg/55/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/jujusvg/56/
Test PASSed.

**Comment by hatched** [comment]
:+1: 

Instead of duplicating the asset so that you don't have a bunch of io calls. Why not import the assets on start? Maybe a followup...sometime :)

**Comment by makyo** [comment]
@hatched this is a library, not a service.  As such, there is no 'start'.

**Comment by makyo** [comment]
updated GUI to QA against:
juju_admin

**Comment by hatched** [comment]
QA Issue:

When using the django bundle (https://jujucharms.com/django/2) the subordinate was shown as a full service.

**Comment by hatched** [comment]
as discussed that subordinate information isn't available to the lib so...QA :ok: YAY! :)

**Comment by makyo** [comment]
Sweet, TY! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/jujusvg-merge

**Pull request merged**